.github/workflows/cypress.yml

```
@@ -208,9 +221,19 @@ jobs:
         HUB_TOKEN=`curl -s -u admin:admin -d '' http://localhost:8002/api/galaxy/v3/auth/token/ | jq -r .token`
         curl -s -H "Authorization: Token $HUB_TOKEN" http://localhost:8002/api/galaxy/ | jq
 
+    - name: "Check if e2e contains only dirs in matrix test array"
+      working-directory: 'ansible-hub-ui'
+      run: |
+        diff -Naur <(ls test/cypress/e2e) <(yq '.jobs.cypress.strategy.matrix.test[]' .github/workflows/cypress.yml | sort)
+
     - name: "Run cypress"
       working-directory: 'ansible-hub-ui/test'
       run: |
+        dir=${{matrix.test}}
+        pattern="s/\*\*/$dir/"
+        sed -i "$pattern" cypress.config.js
```

```suggestion
        sed -i '/specPattern:/s/\*\*/${{matrix.test}}/' cypress.config.js
```

`**` may be present on more than just this line, let's match by the key name first